Model/Service/Product/CachingProductService.php

```diff
@@ -86,7 +92,10 @@ public function getProduct(ProductInterface $product, StoreInterface $store)
             if ($nostoProduct === null) {
                 $nostoProduct = $this->defaultProductService->getProduct($product, $store);
                 if ($nostoProduct !== null) {
-                    $this->nostoCacheService->save($nostoProduct, $store);
```

Comment:
Declare the ttl as a constructor param in CacheService, so there's no need to duplicate it's value on DI.

Reply:
Done